### ISSUE TITLE
Is .nupkg supported for signing?

### ISSUE THREAD

**Issue by filfreire** [SSLcom/esigner-codesign#27]
Hello @bayrakmustafa, as the title says, at the moment we supposedly can EV code-sign .exe files. Can we also code-sign .nupkg files with this GitHub action?

**Comment by pid1** [comment]
@filfreire Though the filetypes in the [README](https://github.com/SSLcom/esigner-codesign/blob/develop/README.md?plain=1#L32) don't include it, [the docs](https://www.ssl.com/guide/esigner-supported-file-types/#ftoc-heading-1) indicate that nuget should be supported. There's a [specific page for nupkg](https://www.ssl.com/how-to/code-signing-nuget-packages-with-esigner/).

The Action itself doesn't perform a file extension check.

You can test this out in the ssl.com sandbox environment without needing to purchase an EV cert first.

**Comment by Bioblaze** [comment]
did you guys get this working?

**Comment by bayrakmustafa** [comment]
@filfreire Yes, it supports nupkg signing. I also added an example nupkg signing at https://github.com/SSLcom/esigner-codesign/blob/develop/.github/workflows/test.yml#L302.
